### DISCUSSION TITLE
How to navigate the app - stuck in last opened project

### DISCUSSION THREAD

**Discussion by kaddison9768** [opengisch/QField#4986] [Q&A]
I toggled on the 'automatically open last project' option. Now I need to get back to the 'main menu' I think the one called File Selector.  But since it auto opens the last map now and the UI design is so excessively stripped down it doesn't appear possible to go back. There has to be a way but I'm out of ideas for where it could possibly be?

**Comment by suricactus** [comment]
If you long press the top-left hamburger menu, you can press the "home" icon (first on the first row). If you use older QField version, you can still find it in the menu with the same icon.
![image](https://github.com/opengisch/QField/assets/2820439/df396df4-9142-48d2-ad3f-85208ac36f17)
